src/main/java/Entities/Schedule.java

```diff
@@ -5,9 +5,10 @@
 public class Schedule {
 
     // todo: make a toString method for Entities.Event to make this work better
-    private HashMap<String, HashMap<String, String>> schedule;  // day, start hour, event name
+    //NOTE: had to make schedule public to access it in ScheduleEditor
+    public HashMap<String, HashMap<String, String>> schedule;  // day, start hour, event name
 
-    public Schedule() {
+    public void schedule() {
```

Comment:
this is supposed to be a constructor, the original public Schedule() was correct, i think?

Reply:
Sorry yeah I'm not sure why this change happened! It was supposed to be Schedule()